onto_spread_ed/release/MergeReleaseStep.py

```
@@ -10,19 +16,52 @@ class MergeReleaseStep(ReleaseStep):
-        for k, file in files:
             ontologies = [self._local_name(n.file) for n in file.sources if n.file.endswith(".owl")]
             version_iri = file.target.iri + "/" + datetime.utcnow().strftime("%Y-%m-%d")
             result += builder.merge_ontologies(ontologies, self._local_name(file.target.file), file.target.iri,
                                                version_iri,
                                                file.target.ontology_annotations)
 
+            self.store_target_artifact(file)
+
+            self._raise_if_canceled()
+
+        for k, file in collapse_imports:
```

Ruff

Loop control variable `k` not used within loop body

Rename unused `k` to `_k`

(B007)